core/domain/stats_jobs_one_off.py

```diff
@@ -16,16 +16,32 @@
 
 import ast
 import collections
+import datetime
+import itertools
+import os
+import random
+import re
+import traceback
 
 from core import jobs
+from core.domain import exp_domain
+from core.domain import exp_services
+from core.domain import interaction_registry
 from core.domain import stats_jobs_continuous
+from core.domain import stats_domain
+from core.domain import stats_services
 from core.platform import models
-(base_models, stats_models, exp_models,) = models.Registry.import_models([
+from extensions.interactions import base
+from extensions.objects.models import objects
+
+import utils
+
+(base_models, stats_models, exp_models) = models.Registry.import_models([
     models.NAMES.base_model, models.NAMES.statistics, models.NAMES.exploration
 ])
-transaction_services = models.Registry.import_transaction_services()
 
 
+# pylint: disable=W0123
```

Comment:
I think this was the warning against using eval() :) Removed it since we aren't using eval anymore.